pyart/core/__init__.py

```diff
@@ -27,7 +27,8 @@
 
 """
 
-from .radar import Radar, is_vpt, to_vpt, join_radar
+from .radar import Radar
 from .grid import Grid
+from ..util.radar_utils import is_vpt, to_vpt
```

Comment:
These should be wrapped using the _depreciated_alias function, see pyart.graph.common for an example.